apps/meteor/app/lib/server/functions/saveUser.js

```diff
@@ -382,12 +383,18 @@ export const saveUser = async function (userId, userData) {
 		await setEmail(userData._id, userData.email, shouldSendVerificationEmailToUser);
 	}
 
-	if (
-		userData.password &&
-		userData.password.trim() &&
-		(await hasPermissionAsync(userId, 'edit-other-user-password')) &&
-		passwordPolicy.validate(userData.password)
-	) {
+	const validPassword = () => {
+		try {
+			passwordPolicy.validate(userData.password);
+		} catch (err) {
+			if (err instanceof PasswordPolicyError) {
+				throw new Meteor.Error(err.error, err.message, err.reasons);
+			}
+			throw err;
+		}
```

Comment:
should, instead of converting to a MeteorError, making frontend not expecting a meteorerror, not be the way? idk maybe that's a lot of work idk, just curious.

Reply:
Hello there! I think the frontend uses that to create the `ToastBar` warnings, but I'm not sure hahah
since this was the previous behavior, perhaps we can keep it like that, and change in another PR. What do you think?